DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -29,6 +30,20 @@ a.delete-button:active {
   background-color: #6b140e;
 }
 
+button.delete-button {
+  background-color: #d5281b;
+  box-shadow: 0 4px 0 #6b140e;
+  margin-bottom: 16px !important;
+}
+
+button.delete-button:hover {
+  background-color: #aa2016;
+}
+
+button.delete-button:active {
+  background-color: #6b140e;
+}

```

Comment:
Could we potentially commonise these styles with the a.delete-button? Also, I think the #d5281b colour is the nhsuk-error-colour:
https://service-manual.nhs.uk/design-system/styles/colour